api/management/api.go

```diff
@@ -298,15 +298,23 @@ type IdentityProviderRepresentation struct {
 type RequiredAction string
 
 func defaultString(actual *string, defaultValue string) *string {
```

Comment:
This method is a bit strange in its behavior. The output might be a reference to the initial string or not. I have a bad feeling about it.

Reply:
That's the same defaultString mechanism as the one in Apache common lang 3